src/Distribution/Server/Pages/Package.hs

```diff
@@ -95,7 +95,7 @@ downloadSection PackageRender{..} =
             [ anchor ! [href cabalURL] << "Package description" ]
           (True,Nothing) ->
             [ anchor ! [href cabalURL] << "Package description"
-            , toHtml " (as included in the package)"
+            , toHtml " (as included in the package as *.cabal file)"
```

Comment:
this isn't great grammar. "as included in the package cabal file" would be fine.